remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -53,64 +54,80 @@ else
     ENABLE_AUTO_SHUTDOWN="$(echo -e "${13}" | tr -d '[:space:]' | tr '[:upper:]' '[:lower:]')"
     # the fourteenth argument is the idle timer in minutes for auto-shutdown
     AUTO_SHUTDOWN_IDLE_TIMER="${14}"
+    # the fifteenth argument is the Binaries location
+    BINARY_LOCATION="${15}"
 fi
 
 update_kernel_dkms()
 {
-  sudo yum -y update
-  sudo yum -y install kernel-devel
-  sudo rpm -Uvh --quiet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-  sudo yum -y install dkms
+    echo "-->Updating" | tee -a "$INST_LOG_FILE"
+    sudo yum -y update
+    echo "-->Installing kernel-devel and dkms" | tee -a "$INST_LOG_FILE"
+    sudo yum -y install kernel-devel
+    sudo rpm -Uvh --quiet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    sudo yum -y install dkms
```

Comment:
I feel like we should check for success/failure for these package installations, since there isn't really much point in continuing if they fail...